pyproject.toml

```diff
@@ -19,7 +19,9 @@ psycopg2 = "^2.9.7"
 lxml = "^4.9.3"
 langserve = {extras = ["server"], version = "^0.0.34"}
 voyageai = "^0.1.4"
-
+faiss-cpu = "1.8.0"
```

Comment:
Is the addition of `faiss-cpu` required? I may have missed it, but I don't think I've it imported

Reply:
Good catch, it's needed for another PR. I might have included it by mistake. It's although a "silent" requirement. More on this in my next PR